src/components/TextField/tests/TextField.test.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import {mountWithAppProvider, findByTestID} from 'test-utilities/legacy';
```

Comment:
Any chance you can use `react-testing`? We're trying not to use the legacy stuff anymore and *eventually* migrate our old tests away